cabal.project.validate.libonly

```diff
@@ -1,28 +1,11 @@
-packages: Cabal-syntax/
-packages: Cabal/
-packages: cabal-testsuite/
-packages: Cabal-QuickCheck/
-packages: Cabal-tree-diff
-packages: Cabal-described
-packages: Cabal-tests
+import: project-cabal/ghc-options.config
+import: project-cabal/pkgs/cabal.config
+import: project-cabal/pkgs/tests.config
+import: project-cabal/constraints.config
 
 tests: True
-
 write-ghc-environment-files: never
 
--- avoiding extra dependencies
-constraints: rere -rere-cfg
-constraints: these -assoc
-
-program-options
-  ghc-options: -fno-ignore-asserts
-
-package Cabal-syntax
-  ghc-options: -Werror
-package Cabal
-  ghc-options: -Werror
-package cabal-testsuite
-  ghc-options: -Werror
-
--- https://github.com/haskell-hvr/cryptohash-sha256/issues/12
-allow-newer: cryptohash-sha256:base
+-- REVIEW: Is the exception for cryptohash-sha256 still needed?
+-- SEE: https://github.com/haskell-hvr/cryptohash-sha256/issues/12
+allow-newer: cryptohash-sha256:base
```

Comment:
Probably not. Would it be simple to check?

Reply:
Only needed for `ghc-9.0.*` and `base-4.15.*`, from [ghc-9.0 wiki](https://gitlab.haskell.org/ghc/ghc/-/wikis/migration/9.0#withforeignptr).

Reply:
But any other library could be likewise affected. Is `ghc-9.0.*` as a whole deprecated?